codec-http/src/main/java/io/netty/handler/codec/http/cors/CorsHandler.java

```diff
@@ -205,8 +208,10 @@ public void write(final ChannelHandlerContext ctx, final Object msg, final Chann
     }
 
     private static void forbidden(final ChannelHandlerContext ctx, final HttpRequest request) {
+        HttpResponse response = new DefaultFullHttpResponse(request.protocolVersion(), FORBIDDEN);
+        response.headers().set(HttpHeaderNames.CONTENT_LENGTH, 0);
```

Comment:
`HtpHeaderValues.ZERO`

Reply:
No problem, I changed that in commit ffd190de2. I squashed commits too so there is a single commit message and reverting is easy. If I shouldn't do that let me know.